code/modules/mob/living/carbon/xenomorph/Powers.dm

```diff
@@ -1954,6 +1944,7 @@
 
 	second_wind_delay = world.time + cooldown
 
+	// since this is always 60 seconds or more, do not turn this into a callback.
```

Comment:
I personally prefer just not having any spawns but @ninjanomnom should look at this first and decide probably

Reply:
arent they both O(logn) for 60 seconds or more?